### PR TITLE
Ultica discord submissions 10 08

### PR DESCRIPTION
**Pull request by acepleiades**
#### Summary
Ultica "discord submissions 10 08"

#### Content of the change
By vetal_l:
- Animal skin tent sprites
- Quarterstaves ground sprites
- Tin cans

By sw_cwya:
- Gideon Bible
- KJV Bible
- Digging stick
- Phone book

### CODE REVIEW

**Comment by acepleiades** [comment]
discord 10 08

**Comment by acepleiades** [comment]
butcher
It works

**Comment by acepleiades** [comment]
new stuff
The newer additions also work

File: gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_door/f_skin_door.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+{
+    "id": [ "f_skin_door", "f_large_skin_door" ],
+    "fg": "f_skin_door",
+    "bg": ""
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_door/f_skin_door.json:5]:
```suggestion
    "fg": "f_skin_door"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_door/f_skin_door.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+},
+{
+    "id": [ "f_skin_door_o", "f_large_skin_door_o" ],
+    "fg": "f_skin_door_o",
+    "bg": ""
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_door/f_skin_door.json:10]:
```suggestion
    "fg": "f_skin_door_o"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_wall/f_skin_wall.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{
+  "id": [ "f_skin_wall", "f_large_skin_wall" ],
+  "multitile": true,
+  "fg": "f_skin_wall_unconnected",
+  "bg": "",
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_wall/f_skin_wall.json:5]:
```suggestion
  "fg": "f_skin_wall_unconnected",
  "multitile": true,
```

File: gfx/UltimateCataclysm/pngs_small_20x20/items/by_sr_cwya/digging_stick.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "digging_stick",
+  "fg": "digging_stick",
+  "bg": ""
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_small_20x20/items/by_sr_cwya/digging_stick.json:4]:
```suggestion
  "fg": "digging_stick"
```

File: gfx/UltimateCataclysm/pngs_small_20x20/items/by_vetal_l/quarterstaff.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "id": "i_staff",
+    "fg": "ironshod_quarterstaff",
+    "bg": ""
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_small_20x20/items/by_vetal_l/quarterstaff.json:5]:
```suggestion
    "fg": "ironshod_quarterstaff"
```

File: gfx/UltimateCataclysm/pngs_small_20x20/items/by_vetal_l/quarterstaff.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  {
+    "id": "shock_staff",
+    "fg": "powered_quarterstaff",
+    "bg": ""
+  },
+  {
+    "id": "q_staff",
+    "fg": "quarterstaff",
+    "bg": ""
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_small_20x20/items/by_vetal_l/quarterstaff.json:15]:
```suggestion
    "fg": "powered_quarterstaff"
  },
  {
    "id": "q_staff",
    "fg": "quarterstaff"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_wall/f_skin_wall.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        "f_skin_wall_end_piece_n",
+        "f_skin_wall_end_piece_w",
+        "f_skin_wall_end_piece_s",
+        "f_skin_wall_end_piece_e"
+      ]
+    },
+    {
+      "bg": "",
+      "id": "unconnected",
+      "fg": "f_skin_wall_unconnected"
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_skin_wall/f_skin_wall.json:53]:
```suggestion
      "fg": "f_skin_wall_center"
    },
    {
      "id": "corner",
      "fg": [
        "f_skin_wall_corner_nw",
        "f_skin_wall_corner_sw",
        "f_skin_wall_corner_se",
        "f_skin_wall_corner_ne"
      ]
    },
    {
      "id": "t_connection",
      "fg": [
        "f_skin_wall_t_connection_n",
        "f_skin_wall_t_connection_w",
        "f_skin_wall_t_connection_s",
        "f_skin_wall_t_connection_e"
      ]
    },
    {
      "id": "edge",
      "fg": [
        "f_skin_wall_edge_ns",
        "f_skin_wall_edge_ew"
      ]
    },
    {
      "id": "end_piece",
      "fg": [
        "f_skin_wall_end_piece_n",
        "f_skin_wall_end_piece_w",
        "f_skin_wall_end_piece_s",
        "f_skin_wall_end_piece_e"
      ]
    },
    {
      "id": "unconnected",
      "fg": "f_skin_wall_unconnected"
```

**Review by int-ua — COMMENTED** [review]
I'll try making the commit myself

**Pull request merged**